### PR TITLE
`Dex` UI updates

### LINKED ISSUE #1075: `DEX` Update UI to focus on Primary Tokens
#### What would you like to be added:
**Table Columns (From L to R)**

1. Liquidity Pair (Example: dBTC - DFI)
    - Primary pair (dBTC) to be bold
    - Secondary pair token (DFI) to be greyed out
2. Primary Token Price (USD)
3. Volume (24H)
4. Total Liquidity with info-tip (similar to APR)
     - To include hover box with the liquidity breakdown (Token Amount **Not USD**)
5. APR

/triage accepted
/area dex ui-ux
/assign @siradji

### PR DESCRIPTION
**Pull request by siradji**
#### What kind of PR is this?:

/kind feature

#### Which issue(s) does this PR fixes?:

Fixes #1075

#### Sample Links & Screenshots:

Link: 
<details>
<summary>Desktop Screenshot</summary>

</details>
<details>
<summary>Mobile Screenshot</summary>

</details>

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on multiple web browsers
- [ ] Tested responsiveness (e.g, iPhone, iPad, Desktop)
- [ ] No console errors
- [ ] Unit tests*
- [ ] Added e2e tests*

### CODE REVIEW

**Comment by nichellekoh** [comment]
Screenshot 2022-03-23 at 11 00 40 AM


@siradji Issue with responsiveness at this point, where APR is blocked out

File: src/pages/vaults/hooks/TokenPrice.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export function useTokenPrice (): DexTokenPrice {
     // active price for walletTokens based on USDT
     const arbitraryPoolPair = getArbitraryPoolPair(symbol, 'USDT')
 
-    if (symbol === 'USDT') {
+    if (symbol === 'USDT' || symbol === 'USDC' || symbol === 'DUSD') {
```
Comment (joeldavidw) [inline src/pages/vaults/hooks/TokenPrice.ts:50]:
```suggestion
    if (symbol === 'USDT') {
```

File: src/pages/dex/_components/PoolPairsCards.tsx [hunk 2]
```diff
@@ -89,42 +90,47 @@ export function PoolPairsCards ({ poolPairs }: { poolPairs: PoolPairData[] }): J
...
+            primaryTextClassName='font-bold'
+            secondaryTextClassName='text-gray-400'
           />
         </div>
       </CardList.Header>
 
       <CardList.List>
         <CardList.ListItem
-          title='Total Liquidity'
+          title='Token Price (USD)'
```
Comment (joeldavidw) [inline src/pages/dex/_components/PoolPairsCards.tsx:112]:
```suggestion
          title='Primary Token Price (USD)'
```

File: src/pages/dex/_components/PoolPairsTable.tsx [hunk 3]
```diff
@@ -38,24 +39,23 @@ export function PoolPairsTable ({ poolPairs }: { poolPairs: PoolPairData[] }): J
     <OverflowTable>
       <OverflowTable.Header>
         <OverflowTable.Head title='Pair' />
-        <OverflowTable.Head title='Total Liquidity' alignRight>
+        <OverflowTable.Head title='Token Price (USD)' alignRight />
```
Comment (joeldavidw) [inline src/pages/dex/_components/PoolPairsTable.tsx:42]:
```suggestion
        <OverflowTable.Head title='Primary Token Price (USD)' alignRight />
```

File: src/pages/vaults/hooks/TokenPrice.ts (hunk 1 above)
Comment (joeldavidw) [reply src/pages/vaults/hooks/TokenPrice.ts:50]: Informed you wrongly.

File: src/components/commons/token/PoolPairSymbolLocal.tsx [hunk 4]
```diff
@@ -15,6 +15,8 @@ interface PoolPairSymbolProps {
   symbolMarginClassName: string
   textClassName: string
   testId?: string
+  primaryTextClassName?: string
+  secondaryTextClassName?: string
```
Comment (joeldavidw) [inline src/components/commons/token/PoolPairSymbolLocal.tsx:19]: Should do similar for symbols. Primary should be larger than secondary.

**Review by joeldavidw — CHANGES_REQUESTED** [review]

File: src/pages/vaults/hooks/TokenPrice.ts (hunk 1 above)
Comment (siradji) [reply src/pages/vaults/hooks/TokenPrice.ts:50]: Yes `USDC` can do below one and we can omit it here but what about `DUSD`? why should we remove it from here?

Comment (joeldavidw) [reply src/pages/vaults/hooks/TokenPrice.ts:50]: Because right now DUSD is not $1 yet. If we fixed it at $1 here the values will be slightly off.

Comment (joeldavidw) [reply src/pages/vaults/hooks/TokenPrice.ts:50]: The calculations here are based off USDT because its the "most stable" and for simplicity sake.

Comment (siradji) [reply src/pages/vaults/hooks/TokenPrice.ts:50]: ignore the comment. i can see `dusd` can go below `1`

File: src/pages/dex/_components/PoolPairsCards.tsx [hunk 5]
```diff
@@ -89,42 +90,47 @@ export function PoolPairsCards ({ poolPairs }: { poolPairs: PoolPairData[] }): J
...
           <PoolPairSymbolLocal
-            tokenA={poolPair.tokenA} tokenB={poolPair.tokenB} symbolSizeClassName='h-6 w-6'
-            symbolMarginClassName='ml-4' textClassName='ml-12 font-medium'
+            tokenA={poolPair.tokenA}
+            tokenB={poolPair.tokenB}
+            primarySymbolClassName='h-6 w-6'
+            secondarySymbolClassName='ml-5 h-4 w-4'
+            textClassName='ml-12'
+            primaryTextClassName='font-bold'
+            secondaryTextClassName='text-gray-400'
```
Comment (joeldavidw) [inline src/pages/dex/_components/PoolPairsCards.tsx:105]:
```suggestion
            primarySymbolClassName='h-7 w-7'
            secondarySymbolClassName='ml-5 h-6 w-6'
            textClassName='ml-12'
            primaryTextClassName='font-medium'
            secondaryTextClassName='text-gray-400'
```

**Review by joeldavidw — CHANGES_REQUESTED** [review]

**Review by nichellekoh — APPROVED** [review]

**Review by joeldavidw — APPROVED** [review]

**Pull request merged**